.eslintrc.cjs

```diff
@@ -29,6 +29,7 @@ module.exports = {
     eqeqeq: [ 'error', 'smart' ],
     'implicit-arrow-linebreak': [ 'error', 'beside' ],
     'import/newline-after-import': 'error',
+    'import/no-useless-path-segments': [ 'error' ],
```

Comment:
For performance wise, `no-useless-path-segments` represents about 3.7% of total eslint time. Not a lot but significant enough to ask for a removal of this module that looks convenient for a one shot PR, but not significant enough, in my opinion, at every single linting (since it fixed 5 errors in years of development)

Reply:
done in d6da67f36